Content.Server/Access/Systems/IdCardConsoleSystem.cs

```diff
@@ -37,6 +48,11 @@ public override void Initialize()
         SubscribeLocalEvent<IdCardConsoleComponent, ComponentStartup>(UpdateUserInterface);
         SubscribeLocalEvent<IdCardConsoleComponent, EntInsertedIntoContainerMessage>(UpdateUserInterface);
         SubscribeLocalEvent<IdCardConsoleComponent, EntRemovedFromContainerMessage>(UpdateUserInterface);
+
+        SubscribeLocalEvent<IdCardConsoleComponent, DamageChangedEvent>(OnDamageChanged);
+
+        // Intercept the event before anyone can do anything with it!
+        SubscribeLocalEvent<IdCardConsoleComponent, MachineDeconstructedEvent>(OnMachineDeconstructed, before: new[] { typeof(EmptyOnMachineDeconstructSystem), typeof(ItemSlotsSystem) });
```

Comment:
Can use a list expression to save like four characters :)
```suggestion
        SubscribeLocalEvent<IdCardConsoleComponent, MachineDeconstructedEvent>(OnMachineDeconstructed, before: [typeof(EmptyOnMachineDeconstructSystem), typeof(ItemSlotsSystem)]);
```

Reply:
I saw you do this in your other PR and its a lot nicer!